x-pack/solutions/observability/packages/utils_server/es/storage/index_adapter/index.ts

```diff
@@ -446,6 +448,22 @@ export class StorageIndexAdapter<TStorageSettings extends IndexStorageSettings>
     });
   };
 
+  private deleteByQuery: StorageClientDeleteByQuery = async ({
```

Comment:
I was thinking we would have a destroy() or reset() method that basically clears up all resources, including templates, to get a clean slate, WDYT?

Reply:
That was the discussion point mentioned the description - happy to rewrite to that instead.

Reply:
sorry, went straight to the files! I think that would be the ideal outcome